deploy.py

```diff
@@ -24,6 +24,12 @@ def build_user_image(image_name, commit_range=None, push=False):
             print("user-image not touched, not building")
             return
 
+    # Attempt to improve relability of pip installs:
+    # https://github.com/travis-ci/travis-ci/issues/2389'''
+    subprocess.check_call([
```

Comment:
This is travis specific, and should hence be in the travis specific `travis/travis-script.bash` file. `deploy.py` should be able to run anywhere.